apps/playground/src/app/modules/components/badge-module/badge-module.component.html

```diff
@@ -1,82 +1,117 @@
-<section class="module-description">
-  <h2>Description</h2>
-  <p>Some description</p>
+<section class='module-description'>
+  <h2>DsfrBadgeComponent</h2>
+  <p>
+    The <code>DsfrBadgeComponent</code> purpose is to represent the badge
```

Comment:
`to represent a badge, meaning, a visual clue as to the state of an element in your user interface.`